data_processors/pipeline/tools/liborca.py

```diff
@@ -134,3 +135,19 @@ def get_sample_names_from_samplesheet(gds_volume: str, samplesheet_path: str) ->
     logger.info(f"Extracted sample names: {sample_names}")
 
     return list(sample_names)
+
+
+def get_subject_id_from_libary_id(library_id):
```

Comment:
This goes to service layer, yes. `data_processors/pipeline/services/metadata_srv.py`

Reply:
Fixed with 0d259b3